### PR TITLE
Add  FLCO-FLYCOLORF7_AIO_HD.config 

### PR DESCRIPTION
**Pull request by tobeornottobe81**
Add  FLCO-FLYCOLORF7_AIO_HD.config

### CODE REVIEW

File: configs/default/FLCO-FLYCOLORF7MINI.config [hunk 1]
```diff
@@ -1,12 +1,24 @@
 # Betaflight / STM32F7X2 (S7X2) 4.2.0 Jun 14 2020 / 03:05:26 (8f2d21460) MSP API: 1.43
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+#define USE_GYRO_SPI_MPU6500
+#define USE_ACC_SPI_MPU6500
+#define USE_FLASH_W25Q128FV
+=======
+=======
+>>>>>>> 6cc6e02698e87cd9321918d463f8c84b249d858f
```
Comment (haslinghuis) [inline configs/default/FLCO-FLYCOLORF7MINI.config:10]: This needs cleanup

File: configs/default/FLCO-FLYCOLORF7MINI.config [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
 #define USE_GYRO_SPI_MPU6500
 #define USE_ACC
 #define USE_ACC_SPI_MPU6500
 #define USE_FLASH
 #define USE_FLASH_W25Q128FV
 #define USE_BARO
+<<<<<<< HEAD
+>>>>>>> 6cc6e02698e87cd9321918d463f8c84b249d858f
+=======
+>>>>>>> 6cc6e02698e87cd9321918d463f8c84b249d858f
```
Comment (haslinghuis) [inline configs/default/FLCO-FLYCOLORF7MINI.config:21]: Same

**Comment by tobeornottobe81** [comment]
Thank you, I have corrected them. Please check.

File: configs/default/FLCO-FLYCOLORF7MINI.config (hunk 1 above)
Comment (tobeornottobe81) [reply configs/default/FLCO-FLYCOLORF7MINI.config]: Okay, it has been cleared.

File: configs/default/FLCO-FLYCOLORF7MINI.config (hunk 2 above)
Comment (tobeornottobe81) [reply configs/default/FLCO-FLYCOLORF7MINI.config]: than you.

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**